### PR TITLE
Add cs.json, updated ro.json, crowdin files.

### PR DESCRIPTION
**Pull request by camilavcoutinho**
Copy 1.0.4 - Copies the full 1.0.4 folder and rename it 1.0.5.

Added czech translations and updated romanian translations.

### CODE REVIEW

File: dist/self-service-reasons/1.0.5/cancellation-reasons_ro.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "formular anulare",
```
Comment (dvm1984) [inline dist/self-service-reasons/1.0.5/cancellation-reasons_ro.json:2]: does this matter? it stayed "cancelForm" for the other languages @camilavcoutinho

File: dist/self-service-reasons/1.0.5/cancellation-reasons_it.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "cancellaModulo",
```
Comment (dvm1984) [inline dist/self-service-reasons/1.0.5/cancellation-reasons_it.json:2]: why "cancelForm" in other languages? impact? @camilavcoutinho

File: dist/self-service-reasons/1.0.2/cancellation-reasons_nl.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "annuleerForm",
```
Comment (dvm1984) [inline dist/self-service-reasons/1.0.2/cancellation-reasons_nl.json:2]: confirm that name needs to change for other languages as well or change to "cancelForm"

File: dist/self-service-reasons/1.0.2/cancellation-reasons_fr.json [hunk 4]
```diff
@@ -1,36 +1,36 @@
 {
-  "name": "cancelForm",
-  "locale": "fr",
+  "name": "Formulaire d’annulation",
```
Comment (kevinch) [inline dist/self-service-reasons/1.0.2/cancellation-reasons_fr.json:2]: Not sure this should be translated given the english value `cancelForm`

File: dist/self-service-reasons/1.0.2/cancellation-reasons_fr.json [hunk 5]
```diff
@@ -1,36 +1,36 @@
...
+  "locale": "en",
```
Comment (kevinch) [inline dist/self-service-reasons/1.0.2/cancellation-reasons_fr.json:3]: Shouldn't it be fr?

File: dist/self-service-reasons/1.0.2/cancellation-reasons_nl.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  "locale": "en",
```
Comment (kevinch) [inline dist/self-service-reasons/1.0.2/cancellation-reasons_nl.json:3]: Shouldn't it be nl?

File: dist/self-service-reasons/1.0.2/replacement-reasons_nl.json [hunk 7]
```diff
@@ -0,0 +1,95 @@
+{
+  "name": "sacForm",
+  "locale": "en",
```
Comment (kevinch) [inline dist/self-service-reasons/1.0.2/replacement-reasons_nl.json:3]: Idem.

**Review by dvm1984 — APPROVED** [review]

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**